localstack-core/localstack/services/lambda_/invocation/counting_service.py

```diff
@@ -125,6 +128,24 @@ def get_invocation_lease(
         unqualified_function_arn = function_version.id.unqualified_arn()
         qualified_arn = function_version.id.qualified_arn()
 
+        # Enforce one lease per ARN if the global flag is set
```

Comment:
What global flag should be set?

Reply:
The LDM flag and in the LDM config we have settings for this qualified arn, hence the `if is_lambda_debug_enabled_for(qualified_arn)`